src/saving/serializers/RegistryTypeConverter.cs

```diff
@@ -72,6 +72,9 @@ public RegistryTypeConverter(ISaveContext context) : base(context)
         if (objectType == typeof(Enzyme))
             return Context.Simulation.GetEnzyme(name);
 
+        if (objectType == typeof(DayNightConfiguration))
+            return Context.Simulation.GetDayNightCycleConfiguration();
```

Comment:
This code should be exactly following how `IAutoEvoConfiguration` is done on lines 81-82.